### PR TITLE
[Copy] Changes to French copy for job poster

### LINKED ISSUE #8660: 🔤 French fixes on the job poster
## 🔤 Copy
These 3 changes on the French job poster

image

**hébergement** should be **mesures d’adaptation**

image

**Publication** instead of **publicité**

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the copy has been added or updated appropriately.

- [ ] All 3 French copy issues fixed
- [ ] Observe other places that contain those words/phrases and spin-off issue?

### PR DESCRIPTION
**Pull request by vd1992**
🤖 Resolves #8660

## 👋 Introduction

Makes the three requested changes. 

## 🕵️ Details

Matt dug around a bit as well in the issue comments https://github.com/GCTC-NTGC/gc-digital-talent/issues/8660#issuecomment-1839562248
Location is changed to "lieu" but is translated as "emplacement" elsewhere, noted in https://github.com/GCTC-NTGC/gc-digital-talent/issues/8463#issuecomment-1846260714

## 🧪 Testing

1. Navigate to poster page, switch to French
2. Observe the three requested changes are implemented
3. Two are on the poster view, the third is seen in the application flow

## 📸 Screenshot

image

### CODE REVIEW

**Comment by tristan-orourke** [comment]
Recruitment team has confirmed that they prefer "Lieu" to "Emplacement", including for our other uses of the word. Let's replace "emplacement" with "Lieu" both on the poster, and on the job poster builder.

File: apps/web/src/lang/fr.json
Comment (mnigh) [inline apps/web/src/lang/fr.json:2207]: This string doesn't include punctuation in English, nor should it in French.


## Screenshot

Screen Shot 2023-12-11 at 14 01 41

```suggestion
    "defaultMessage": "Lieu, langue et sécurité",
```

**Review by mnigh — COMMENTED** [review]
The changes for **location** and **accommodation** make sense but I'm still not sure this change goes far enough for **advertisement**.

File: apps/web/src/lang/fr.json
Comment (vd1992) [reply apps/web/src/lang/fr.json:5167]: Do we want to check in this issue's scope or push this for later?

Comment (tristan-orourke) [reply apps/web/src/lang/fr.json:5167]: If we're using "annonce" elsewhere for advertisement, and this is the first time we'd use "publication", let's just change this to "Revenir à l'annonce".

Comment (mnigh) [reply apps/web/src/lang/fr.json:5167]: @tristan-orourke can we change `rwQPZE` from _Informations publicitaires_ to _Informations de l'annonce_ as well?

Comment (vd1992) [reply apps/web/src/lang/fr.json]: Done the first in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8750/commits/482c8f4908765c2f1eb697f004d3525d8701bdf2
And the tentative in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8750/commits/b9f7d4900b4979cc6d8799305489ae280f77f53d

**Review by mnigh — APPROVED** [review]
LGTM 👍 (pending @tristan-orourke's approval on b9f7d4900b4979cc6d8799305489ae280f77f53d)

**Review by tristan-orourke — APPROVED** [review]
https://github.com/GCTC-NTGC/gc-digital-talent/commit/b9f7d4900b4979cc6d8799305489ae280f77f53d seems like a good call to me. 👍

**Pull request merged**
